packages/bpk-component-modal/src/BpkModalV2/BpkModal.tsx

```diff
@@ -38,10 +38,39 @@ export type Props = {
   noFullScreenOnMobile?: boolean;
   onClose: () => void | null;
   padded?: boolean;
+  showHeader: boolean;
```

Comment:
I don't think this should be a required prop (also looking at the description of the PR) 🤔 shouldn't it have a default value which is `true` in which case this prop would simply introduce the option of hiding the header?